server/mediaserver.go

```diff
@@ -227,18 +227,12 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 		s.connectionLock.Unlock()
 		LastManifestID = mid
 
-		//We try to automatically determine the video profile from the RTMP stream.
-		var vProfile ffmpeg.VideoProfile
+		// Build the source video profile from the RTMP stream.
 		resolution := fmt.Sprintf("%vx%v", rtmpStrm.Width(), rtmpStrm.Height())
-		for _, vp := range ffmpeg.VideoProfileLookup {
-			if vp.Resolution == resolution {
-				vProfile = vp
-				break
-			}
-		}
-		if vProfile.Name == "" {
-			vProfile = ffmpeg.P720p30fps16x9
-			glog.V(common.SHORT).Infof("Cannot automatically detect the video profile - setting it to %v", vProfile)
+		vProfile := ffmpeg.VideoProfile{
+			Name:       "source",
+			Resolution: resolution,
+			Bitrate:    "4000k", // Fix this
```

Comment:
Where does 400k come from? I assume it might be there as a placeholder, until we build a function to autodetect bitrate?

Reply:
Exactly, yes. It was taken from the "old" 720p default, https://github.com/livepeer/lpms/blob/master/ffmpeg/videoprofile.go#L31 . Mostly inaccurate until we have a better way of returning the estimated bitrate.

Reply:
Awesome! Would we use something like ffprobe for that?

Reply:
The bandwidth indicator in HLS indicates the peak (maximum) bitrate; I suppose we'll need to modify LPMS to return a few metrics which we can use to derive the information (min, max, total, duration, etc). https://tools.ietf.org/html/rfc8216#section-4.3.4.2

Reply:
OooOO cool!